src/Core/Framework/Resources/config/packages/shopware.yaml

```diff
@@ -521,9 +522,12 @@ shopware:
         enable_stock_management: true
 
     usage_data:
+        collection_enabled: true
         gateway:
             dispatch_enabled: true
-            base_uri: 'https://data.shopware.io'
+            entity_dispatch_enabled: true
+            consent_dispatch_enabled: true
+            base_uri: '%env(SHOPWARE_USAGE_DATA_GATEWAY_BASE_URI)%'
```

Comment:
I would not introduce a new environment variable, nobody except us will change it. why you need it, you can do it in your own `config/packages/xxx.yml` file

Reply:
you can introduce that env in the saas template :)

Reply:
Ah thank you! Was not thinking about overriding it there with the value from the env. Will do so! :smile:

Reply:
I reverted this change and will handle that in the SaaS template directly :smile: